python/rikai/torch/data.py

```diff
@@ -346,8 +338,8 @@ def _get_cache_uri(df: "pyspark.sql.DataFrame") -> str:
     cache_uri: str
         The uri to write the DataFrame to
     """
-    # TODO also try to get it from rikai.options once #134 is done
-    from rikai.spark.conf import CONF_SPARK_RIKAI_CACHEURI
-
-    cache_root_uri = df.rdd.ctx.getConf().get(CONF_SPARK_RIKAI_CACHEURI)
+    cache_root_uri = rikai.options.rikai.cache_uri
+    if not cache_root_uri:
+        from rikai.conf import CONF_RIKAI_CACHEURI
+        cache_root_uri = df.rdd.ctx.getConf().get(CONF_RIKAI_CACHEURI)
```

Comment:
does it provide a `get(key, default)` ? 

`rikai.conf` only has dependency to pandas, we do need to do lazy import IIUC.

Reply:
is `/tmp` a reasonable default? I'm a little bit concerned that people would just use it blindly and then fill up the cluster local disk.

Reply:
yea, is there some function to read `TMP_DIR` in python? if so , lets use that to make it work well with devops ?